javacord-api/src/main/java/org/javacord/api/entity/server/Server.java

```diff
@@ -2282,6 +2283,55 @@ default CompletableFuture<Void> banUser(long userId, Duration duration, String r
         return banUser(Long.toUnsignedString(userId), duration, reason);
     }
 
+    /**
+     * Bans the given user from the server.
+     *
+     * @param userIds The ids of the users to ban.
+     * @return A future returning details about the bulk ban.
+     */
+    default CompletableFuture<BulkBanResponse> bulkBanUsers(long... userIds) {
+        return bulkBanUsers(
+                LongStream.of(userIds).boxed().map(Long::toUnsignedString).collect(Collectors.toList()),
+                0, TimeUnit.SECONDS, null);
+    }
+
+    /**
+     * Bans the given user from the server.
+     *
+     * @param userIds The ids of the users to ban.
+     * @return A future returning details about the bulk ban.
+     */
+    default CompletableFuture<BulkBanResponse> bulkBanUsers(String... userIds) {
+        return bulkBanUsers(Arrays.stream(userIds).collect(Collectors.toList()), 0, TimeUnit.SECONDS, null);
```

Comment:
Seems a bit overly complex? :D
```suggestion
        return bulkBanUsers(Arrays.asList(userIds), 0, TimeUnit.SECONDS, null);
```